In [1]:
product = {
  "product": {
    "id": "e9d10e5c6a1e4b3e944fb8fcde5bb883",
    "parentId": None,
    "name": "Premium Leather Jacket",
    "description": "A classic, genuine leather jacket with premium stitching.",
    "productNumber": "SW10001",
    "stock": 45,
    "availableStock": 45,
    "active": True,
    "shippingFree": False,
    "calculatedPrice": {
      "unitPrice": 199.99,
      "quantity": 1,
      "totalPrice": 199.99,
      "calculatedTaxes": [
        {
          "tax": 31.94,
          "taxRate": 19.0,
          "price": 199.99
        }
      ],
      "regulationPrice": None
    },
    "cover": {
      "id": "7a8b9c0d1e2f3g4h5i6j7k8l9m0n1o2p",
      "media": {
        "id": "media_id_12345",
        "url": "https://yourshopdomain.com",
        "alt": "Premium Leather Jacket front view",
        "width": 1200,
        "height": 1200
      }
    },
    "properties": [
      {
        "id": "prop_color_id",
        "name": "Color",
        "group": {
          "id": "group_color_id",
          "name": "Colors"
        },
        "translated": {
          "name": "Black"
        }
      },
      {
        "id": "prop_size_id",
        "name": "Size",
        "group": {
          "id": "group_size_id",
          "name": "Sizes"
        },
        "translated": {
          "name": "L"
        }
      }
    ],
    "manufacturer": {
      "id": "manufacturer_id_100",
      "name": "FashionBrand",
      "link": "https://yourshopdomain.com"
    }
  }
}


In [7]:
p = product["product"]

name = p["name"]

i_d = p["id"]

price = p["calculatedPrice"]["calculatedTaxes"][0]["price"]

tax = p["calculatedPrice"]["calculatedTaxes"][0]["taxRate"]


for prop in p["properties"]:
    colour = prop["name"]
    black = prop["translated"]["name"]

for prop in p["properties"]:
    size = prop["name"]
    size1 = prop["translated"]["name"]

manufacturer = p["manufacturer"]["name"]

print(f"Product Name: {name}")
print(f"Product ID: {i_d}")
print(f"Product Price: {price}")
print(f"Product Taxes: {tax}")
print(f"Product Colour: {colour}")
print(f"Product Size: {size1}")
print(f"Product Manufacturer: {manufacturer}")


Product Name: Premium Leather Jacket
Product ID: e9d10e5c6a1e4b3e944fb8fcde5bb883
Product Price: 199.99
Product Taxes: 19.0
Product Colour: Size
Product Size: L
Product Manufacturer: FashionBrand


- Produkt Name
- Produkt ID
- Produkt price  <-  calculatedTax => price
- Produkt Tax Rate: 19%
- Die Eigenschaften des Produktes : Color = Black, Size = L
- Hersteller Name: FashionBrand